### PR TITLE
Fix/issue 509

### PR DESCRIPTION
**Pull request by iparask**
This is to take care of cases like issue #509. I still to find a proper fix for that one.

### CODE REVIEW

File: src/radical/entk/execman/rp/task_manager.py [hunk 1]
```diff
@@ -313,7 +314,9 @@ def unit_state_cb(unit, state):
 
                     self._advance(task, 'Task', states.SUBMITTING,
                                   mq_channel, '%s-tmgr-to-sync' % self._sid)
-
+                    self._log.debug('Connection State %s channel state %s'
+                                        % (mq_connection.is_open,
+                                           mq_channel.is_open))
```
Comment (andre-merzky) [inline src/radical/entk/execman/rp/task_manager.py:319]: I would suggest to disable that message - it's on a hot code path.  OTOH, it's `DEBUG` level, and we should not encourage to run `DEBUG` as default level either.  Hmm, so probably ok... :-)

**Review by andre-merzky — APPROVED** [review]
LGTM

File: src/radical/entk/appman/appmanager.py [hunk 2]
```diff
@@ -22,6 +22,8 @@
 
 from .wfprocessor  import WFprocessor
 
+pika.connection.Parameters.DEFAULT_HEARTBEAT_INTERVAL = 0
+pika.connection.Parameters.DEFAULT_HEARTBEAT_TIMEOUT = 0
```
Comment (mturilli) [inline src/radical/entk/appman/appmanager.py:27]: Do we want that to be the default behavior for EnTK and RBMQ? Do we have issues only with one user and one machine or is the heartbeat_interval going to be a more general issue? If the former, then should we keep the default as it has been so far and enable setting the interval/timeout to a specific value via env variables?

**Comment by iparask** [comment]
Closing this PR as there is an easy workaround

**Pull request closed without merging**